tss-esapi-sys/README.md

```diff
@@ -18,7 +18,7 @@ must therefore link to all of them at build time.
 
 The paths to the libraries are discovered using `pkg-config` - make sure they
 are discoverable in this way on your system. Our build script looks for 
-`tss2-esys`, `tss2-tctildr` and `tss2-mu`. A minimum version of `3.2.2` is 
+`tss2-esys`, `tss2-tctildr` and `tss2-mu`. A minimum version of `4.0.1` is 
```

Comment:
Should we change [the version we get in our test containers](https://github.com/parallaxsecond/rust-tss-esapi/blob/main/tss-esapi/tests/Dockerfile-ubuntu#L10)?

Reply:
Yes, I realized I have missed updating the version in other places as well.

Reply:
But I am having some issues getting the ```regenerate_bindings``` script to work with  4.0.1

Reply:
Fixed now